lib/kitchen/transport/winrm_file_transfer/remote_zip_file.rb

```diff
@@ -92,6 +93,7 @@ def extract_zip_command
             $shellApplication = new-object -com shell.application
 
             $zipPackage = $shellApplication.NameSpace('#{remote_path}')
+            Remove-Item -Recurse -Force $dest | Out-Null
```

Comment:
Hello!
Looks like Windows 2003 R2 Server API does not handle correctly zip files, where threre are dedicated zip entries, created for the folders.
Initially, the TK was just hanging during the stage of transfering files to the guest. 
With the  '-l debug' option added to TK command like, 
I narrowed down the problem to the stage of unpacking the transfered 'kitchen.zip' archive with all cookbooks and chef staff. The unpacking was hanging on creating some subfolders, while the files in the root folder were unpacked correctly.

I wanted to verify the correctness of the archive and opened it in the  Windows explorer archive viewever, resulting in the following content (i took screenshot of apache cookbook subfolder, packed in kitchen.zip):
![screenshot from 2015-02-12 11 02 14](https://cloud.githubusercontent.com/assets/8427785/6165275/eb0e0cc4-b2a6-11e4-8861-4d9ec9e8c1bd.png)
So the directory entries in the zip files were interpreted by Windows Explorer API as files, though the archive was correctly unpacked, using unzip, 7zip or any other third-party tools.

Afterwards, I took the powershell commands from the 'remote_zip_file' script and tried to perform them manually in the powershell, resulting in the following error, which was causing the TK to hang:
![screenshot from 2015-02-12 11 10 08](https://cloud.githubusercontent.com/assets/8427785/6165370/bb27a41a-b2a7-11e4-8569-37268cca90c1.png)
Tthe archived was extracted correctly by TK on Windows 2003 after removing the directory entries from the zip file, using code:
next if File.directory?(file)